sdk/translation/Azure.AI.Translation.Document/tests/Infrastructure/DocumentTranslationSampleHelper.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Core.Tests.TestFramework;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Sas;
+
+namespace Azure.AI.Translation.Document.Tests
+{
+    internal static class DocumentTranslationSampleHelper
+    {
+        public static DocumentTranslationTestEnvironment TestEnvironment;
+        public static readonly List<TestDocument> oneTestDocuments = new()
+        {
+            new TestDocument("Document1.txt", "First english test document"),
+        };
+
+        private static TestRandom random
+        {
+            get
+            {
+                var csp = new RNGCryptoServiceProvider();
+                var bytes = new byte[4];
+                csp.GetBytes(bytes);
+                return new TestRandom(RecordedTestMode.Live, BitConverter.ToInt32(bytes, 0));
+            }
+        }
```

Comment:
@JoshLove-msft  please review

Reply:
Thank you for looking at my PR, but please do not waste too much time on it as this is not the approach that we will be using. We will most likely be developing in the following PR #23072, which follows a more conventional way for achieving the same goal.